backend/object-management/src/main/kotlin/com/ritense/valtimoplugins/objectmanagement/service/ObjectManagementCrudService.kt

```diff
@@ -69,11 +70,18 @@ class ObjectManagementCrudService(
                 startAt = LocalDate.now()
             )
         )
-        return objectenApiPlugin.createObject(objectRequest).url
+
+        return objectenApiPlugin.objectPatch(objectUrl, objectRequest).url
     }
 
-    fun deleteObject(objectUrl: URI) {
-        return
+    fun deleteObject(
+        objectUrl: String,
+        objectManagementConfigurationId: UUID
+    ) {
+        val objectManagement = getObjectManagement(objectManagementConfigurationId)
+        val objectenApiPlugin = getObjectenApiPlugin(objectManagement.objectenApiPluginConfigurationId)
+        val uri = URI.create(objectUrl)
+        objectenApiPlugin.deleteObject(uri)
```

Comment:
It looks like this `objectManagementCrudService` has many similar methods as the `ObjectManagementFacade`. It would be nice if the `ObjectManagementFacade` can be used to avoid duplication